src/Sensor.cpp

```diff
@@ -9,7 +9,7 @@
  * the flight software being tested can obtain information of the simulated rocket.
  * The classes provide a modular a means of injecting noise and bias along with
  * other inaccuracies to make sensor measurements behave closer to real hardware
- * rather than simply proving gorund truth. 
+ * rather than simply providing gorund truth. 
```

Comment:
Lmao nit: I like how you fixed one typo but didn't see the other typo right beside it.

Reply:
oh woops. fixed now.